videos.html

```diff
@@ -94,14 +102,6 @@ <h1 class="page-header">Videos</h1>
             <iframe width="640" height="360" src="https://www.youtube.com/embed/YfBDDWp58po?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
           </div>
 
-          <h2>Interledger Overview and Examples</h2>
-
-          <p>This video series, from the February 2016 Interledger Workshop, includes an Interledger overview, demos and presentations from various Interledger Community Group members. The first video includes a walk-through of the main Interledger Transport protocol and explains how it provides payment reliability.</p>
-
-          <div class="video-container">
-            <iframe width="640" height="360" src="https://www.youtube.com/embed/UdCxrqP6w3I?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
-          </div>
-
       </div>
```

Comment:
They basically say the same thing, but with a slightly older phrasing. The videos are still available on Youtube, I don' think we need all the videos we've ever done to be available here.